charts/nginx-ingress/templates/_helpers.tpl

```diff
@@ -361,6 +381,16 @@ volumeMounts:
 - mountPath: /var/log/nginx
   name: nginx-log
 {{- end }}
+{{- if .Values.controller.appprotect.v5 }}
+- name: app-protect-bd-config
+  mountPath: /opt/app_protect/bd_config
+- name: app-protect-config
+  mountPath: /opt/app_protect/config
+  # Normally, only the waf-config-mgr should access this dir.
+  # This is here for testing.
```

Comment:
@jjngx Can you confirm if this needs to be removed?

Reply:
yes, please remove